docs/modules/ROOT/pages/how-tos/use-exoscale.adoc

```diff
@@ -0,0 +1,44 @@
+= Use Exoscale
+
+The following steps show how to set up Terraform with Exoscale.
+
+. Set up 3 new API keys in https://portal.exoscale.com[portal.exoscale.com].
+  Two of them are used for the Terraform pipeline.
+  The first key should be created with restricted (compute > `get*`, `list*`, `query*`, `show*` and `validate*`; DNS > `list*` and `get*`) permissions
+  The second key can be created as `unrestricted`.
+  The third key needs the following permissions (this key will be deployed onto the LBs for https://git.vshn.net/appuio-public/ursula[Ursula]):
+  - compute:
+    * `addIpToNic`
+    * `listNics`
+    * `listResourceDetails`
+    * `listVirtualMachines`
+    * `queryAsyncJobResult`
+    * `removeIpFromNic`
+. Configure component parameters.
++
+[source,yaml]
+----
+openshift4_terraform:
+  provider: exoscale
+  terraform_variables:
+    worker_count: 3                         # optional
+    worker_size: Extra-large                # optional
+    rhcos_template: TheTemplateNameForRHCOS
+    base_domain: ${openshift:baseDomain}
+    ignition_ca: |-
+      -----BEGIN CERTIFICATE-----
+      ...
+    ssh_key: ssh-ed25519 AA...
+    bootstrap_bucket: my-bucket
```

Comment:
Is the bucket correct? Or is it like a full endpoint with a reclass parameter e.g. `https://...ch/${cluster:name}` ?

Reply:
Hmm, actually should be the full endpoint, I didn't spot that